cfgov/regulations3k/jinja2/regulations3k/regulations-listing.html

```diff
@@ -0,0 +1,51 @@
+{# ==========================================================================
+
+   Regulation listing
+
+   ==========================================================================
+
+   Description:
+
+   Creates a table of regulations when given:
+
+   regulations:    List of regulation part objects
+
+   heading:        The heading of the regulations list.
+
+   more_regs_url:  A string for the URL of the more regulations link.
+
+   more_regs_text: A string for the text of the more regulations link.
+
+   ========================================================================== #}
+{% if regulations.exists() %}
+<table class="o-table" data-bound="true">
```

Comment:
Doesn't have to hold up this PR, but I wonder if a using an unordered list instead of a table would make more sense here. On one hand, using an `o-table` is nice since it's a standard component and we don't have to make any front-end customizations to get it looking right. On the other, since we're now down to just a single column that has links to each reg (and eventually links to future amendments), I think an unordered list would be more semantic (though it would obviously require some custom styles to match the design).

Reply:
We certainly could do — I leave that up to you and @contolini.

Reply:
But then it might need to be titled "Unordered list of contents" 😛

Reply:
@niqjohnson I think you're right that `<ul>` makes more sense. I bet we can get a list to mimic a table's styles pretty easily. To avoid holding up this PR let's open a separate issue to do it.

Reply:
@contolini, converting the table to an unordered list in a separate PR works for me. Thanks!